machines/README.md

```diff
@@ -0,0 +1,27 @@
+# Vagrantfile for WSO2 Identity Server
+
+This section defines the procedure to execute run Vagrant resources for a setup of WSO2 Identity Server single
+node with Analytics support.<br>
```

Comment:
Aren't we going to have two deployment instances where one would only deploy IS and the other would deploy both IS and Analytics ?

Reply:
No, we decided to simply have one deployment which supports both IS and Analytics.